Applications/Spire/Include/Spire/Ui/DropDownItem.hpp

```diff
@@ -0,0 +1,74 @@
+#ifndef SPIRE_DROP_DOWN_ITEM_HPP
+#define SPIRE_DROP_DOWN_ITEM_HPP
+#include <QImage>
+#include <QWidget>
+#include "Spire/Spire/Spire.hpp"
+#include "Spire/Ui/CustomQtVariants.hpp"
+
+namespace Spire {
+
+  //! Represents an item used in a DropDownList.
+  class DropDownItem : public QWidget {
+    public:
+
+      //! Signals that the item was highlighted with the mouse.
+      /*!
+        \param value The value of the DropDownItem.
+      */
+      using HighlightedSignal = Signal<void (QVariant value)>;
```

Comment:
Any reason to take the variant by copy here and in `SelectedSignal`?

Reply:
The issue I was having was that the items can be removed from the list after they're selected, so DropDownItem's value might not exist after the signal is triggered.